docs/error-codes.md

```diff
@@ -169,4 +169,20 @@ error and warning codes.
 
 #### `IL2028`: Attribute 'attribute' on 'method' doesn't have a required constructor argument.
 
-- The linker found an instance of attribute 'attribute' on 'method' but it lacks a required constructor argument. Linker will ignore this attribute.
\ No newline at end of file
+- The linker found an instance of attribute 'attribute' on 'method' but it lacks a required constructor argument. Linker will ignore this attribute.
+
+#### `IL2029`: There is no xml attribute 'fullname' in xml element '{attributeFullName}'
+
+- The linker found an instance of attribute 'attribute' on 'method' but it lacks a required constructor argument. Linker will ignore this attribute.
```

Comment:
The description doesn't match the title in this case - probably copy/paste error.